discord/abc.py

```diff
@@ -324,7 +326,7 @@ def overwrites_for(self, obj):
         Returns
         ---------
         :class:`PermissionOverwrite`
```

Comment:
Functions/properties that can return None are usually marked `Optional`.
```suggestion
         Optional[:class:`PermissionOverwrite`]
```